File: 백준/Bronze/1934. 최소공배수/최소공배수.py
def lcm(x,y):
    return (x*y)/gcd(x,y)
def gcd(x,y):
    r=1
    while r!=0:
        r=x%y
        x=y;y=r
    return x
testcase=int(input())
a=[0 for col in range(testcase)]; b=[0 for col in range(testcase)]; c=[0 for col in range(testcase)]
for i in range(testcase):
    a[i],b[i]=map(int,input().split())
    c[i]=lcm(a[i],b[i])
for j in range(testcase):
    print(int(c[j]))

File: 백준/Bronze/1934. 최소공배수/최소공배수.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.Arrays;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringBuilder sb = new StringBuilder();
        int t = Integer.parseInt(br.readLine());
        for (int i = 0; i < t; i++) {
            int[] input = Arrays.stream(br.readLine().split(" ")).mapToInt(Integer::parseInt).toArray();
            int a = input[0], b = input[1];
            sb.append(lcm(a, b)).append("\n");
        }
        System.out.println(sb);
    }

    private static int lcm(int a, int b) {
        return a * b / (gcd(a, b));
    }

    private static int gcd(int a, int b) {
        if (b == 0) {
            return a;
        }
        return gcd(b, a % b);
    }
}